src/identity/Identity.ts

```diff
@@ -28,9 +28,43 @@ type BoxPublicKey =
 
 export default class Identity extends PublicIdentity {
   private static ADDITIONAL_ENTROPY_FOR_HASHING = new Uint8Array([1, 2, 3])
+  /**
+   * [STATIC] Generates Mnemonic phrase used to create identities from phrase seed.
+   * @returns `generate`
+   * @example
+   * ```javascript
+   *
+   * Identity.generateMnemonic()
+   * // mnemonic: "coast ugly state lunch repeat step armed goose together pottery bind mention"
+   *
+   * ```
+   */
   public static generateMnemonic() {
     return generate()
   }
+
+  /**
+   * [STATIC] Builds an identity object from a mnemonic string.
+   * @param phraseArg [[BIP39]](https://www.npmjs.com/package/bip39) Mnemonic word phrase.
+   * @returns `Identity`
+   *
+   * @example
+   *
+   * ```javascript
+   * //Build a Mnemonic to use as an argument
+   *
+   * const mnemonic = Identity.generateMnemonic()
+   * // "mnemonic: coast ugly state lunch repeat step armed goose together pottery bind mention"
+   *
+   * const testIdentity = Identity.buildFromMnemonic(mnemonic)
+   * // testIdentity: Identity {
```

Comment:
Maybe here add "Output:", this will clarify what this comment is

Reply:
An output added, maybe like this?